### PR TITLE
SISRP-16653 - My Finances Page Gap

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-16653

### CODE REVIEW

File: src/assets/templates/myfinances.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h1 class="cc-heading-page-title">My Finances</h1>
       <div data-ng-include="'widgets/finaid_activity.html'" data-ng-if="!api.user.profile.isApplicantOnly"></div>
     </div>
 
-    <div class="medium-4 columns" data-ng-class="{'medium-pull-4':(api.user.profile.isApplicantOnly)}">
+    <div class="medium-4 columns cc-column-last" data-ng-class="{'medium-pull-4':(api.user.profile.isApplicantOnly)}">
```
Comment (christianvuerings) [inline src/assets/templates/myfinances.html]: @dfkaye were you also able to test this for people with only the `applicant` role?

Comment (dfkaye) [inline src/assets/templates/myfinances.html]: @christianvuerings no I wasn't aware of that

Comment (christianvuerings) [inline src/assets/templates/myfinances.html]: @dfkaye ok, would you be able to test that and see whether it looks ok?

**Comment by dfkaye** [comment]
@christianvuerings I've amended this in the conditional classname check -

**Pull request merged**